advanced/Scripts/list.sh

```diff
@@ -50,9 +50,18 @@ EOM
 	exit 1
 }
 
+EscapeRegexp() {
+    # This way we may safely insert an arbitrary
+    # string in our regular expressions
+    echo $* | sed 's/[]\.|$(){}?+*^]/\\&/g' | sed 's/\//\\\//g'
```

Comment:
We have to force this one. I'm not even sure if this particular `sed` can actually be done with double quotes. There is nothing to be expanded here, so codacy's comment doesn't matter here.